x-pack/plugins/security_solution/public/common/components/alert_count_by_status/alert_count_by_rule_by_status.tsx

```diff
@@ -0,0 +1,208 @@
+          [field]: selection,
+        });
+      },
+      [field, selectedStatusesByField, setSelectedStatusesByField]
+    );
+
+    const { items, isLoading, updatedAt } = useAlertCountByRuleByStatus({
+      field,
+      value,
+      queryId,
+      statuses: selectedStatusesByField[field] as Status[],
+      skip: !toggleStatus,
+      signalIndexName,
+    });
+
+    return (
+      <HoverVisibilityContainer show={true} targetClassNames={[INSPECT_BUTTON_CLASS]}>
+        <StyledEuiPanel hasBorder data-test-subj="alertCountByRulePanel">
+          <>
+            <HeaderSection
+              id={queryId}
+              title={i18n.Alerts}
+              titleSize="s"
+              toggleStatus={toggleStatus}
+              toggleQuery={setToggleStatus}
+              subtitle={<LastUpdatedAt updatedAt={updatedAt} isUpdating={isLoading} />}
+            >
+              <MultiSelectPopover
+                title={i18n.Status}
+                allItems={statuses}
+                selectedItems={selectedStatusesByField[field] || ['open']}
+                onSelectedItemsChange={(selectedItems) =>
+                  updateSelection(selectedItems as Status[])
+                }
+              />
+            </HeaderSection>
+
+            {toggleStatus && (
+              <>
+                <EuiBasicTable
+                  className="eui-yScroll"
+                  data-test-subj="alertCountByRuleTable"
+                  columns={columns}
+                  items={items}
+                  loading={isLoading}
+                  noItemsMessage={
+                    <EuiEmptyPrompt title={<h3>{i18n.NO_ALERTS_FOUND}</h3>} titleSize="xs" />
+                  }
+                />
+              </>
+            )}
+          </>
+        </StyledEuiPanel>
+      </HoverVisibilityContainer>
+    );
+  }
+);
+
+AlertCountByRuleByStatus.displayName = 'AlertCountByStatus';
+
+export const getTableColumns: GetTableColumns = ({
+  getAppUrl,
+  navigateTo,
+  openRuleInTimelineWithAdditionalFields,
+}) => [
+  {
+    field: 'ruleName',
+    name: i18n.COLUMN_HEADER_RULE_NAME,
+    'data-test-subj': i18n.COLUMN_HEADER_RULE_NAME,
+    align: 'left',
+    width: '67%',
+    sortable: false,
+    render: (ruleName: string, { uuid }) => {
+      const url = getAppUrl({ deepLinkId: SecurityPageName.rules, path: `id/${uuid}` });
+      return (
+        <EuiToolTip
+          data-test-subj={`${ruleName}-tooltip`}
+          title={i18n.TOOLTIP_TITLE}
+          content={ruleName}
+          anchorClassName="eui-textTruncate"
+        >
+          {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
+          <EuiLink
+            data-test-subj="severityRuleAlertsTable-name"
+            href={url}
+            onClick={(ev?: React.MouseEvent) => {
+              if (ev) {
+                ev.preventDefault();
+              }
+              navigateTo({ url });
+            }}
+          >
+            {ruleName}
+          </EuiLink>
```

Comment:
Why not use `SecuritySolutionLinkAnchor`? and forget about urls, onClick, navigation functions...
```suggestion
          <SecuritySolutionLinkAnchor
            data-test-subj="severityRuleAlertsTable-name"
            deepLinkId={SecurityPageName.rules} 
            path={`id/${uuid}`} 
          >
            {ruleName}
          </SecuritySolutionLinkAnchor>
```

Reply:
updated!